core/src/main/java/org/bitcoinj/core/Transaction.java

```diff
@@ -794,7 +799,7 @@ public String toString(@Nullable AbstractBlockChain chain, @Nullable CharSequenc
         }
         if (!requiresInputs()) {
             // no ins, no outs
-            if (getVersionShort() == 3 && type.isSpecial())
+            if (getVersionShort() == MAX_STANDARD_VERSION && type.isSpecial())
```

Comment:
This should be DIP2_VERSION, since MAX_STANDARD_VERSION may be raised to 4 at a later time.